aten/src/ATen/native/native_functions.yaml

```diff
@@ -5101,6 +5101,11 @@
     SparseCPU, SparseCUDA: resize_as_sparse_
     SparseCsrCPU, SparseCsrCUDA: resize_as_sparse_csr_
 
+- func: zero(Tensor self) -> Tensor
+  variants: function
+  dispatch:
+    CompositeExplicitAutograd: zero
```

Comment:
I don't want this to be `CompositeImplicitAutograd` because I want you to be able to trace out a `zero` call.

@ezyang this should probably go under the new `CompositeNonFunctional` alias key once we have one (name tbd), so functional backends don't actually decompose it.

Reply:
TBH I am confused, what is wrong with this getting traced as `zeros_like`?